frontend/src/metabase/query_builder/components/view/QuestionDataSource.jsx

```diff
@@ -29,7 +39,9 @@ function getDataSourceParts({ question, noLink, subHead, isObjectDetail }) {
   const parts = [];
 
   let query = question.query();
-  if (query instanceof StructuredQuery) {
+  const isStructuredQuery = query instanceof StructuredQuery;
```

Comment:
since we have access to the `question` here, let's go ahead and change this to `question.isStructured()` so we can avoid importing `StructuredQuery`